app/utilities/schema.py

```diff
@@ -20,25 +20,35 @@ def load_schema_from_params(eq_id, form_type, language_code='en'):
     return load_schema_file("{}_{}.json".format(eq_id, form_type), language_code)
 
 
-def load_schema_file(schema_file, language_code='en'):
-    logger.info("loading schema", filename=schema_file, language_code=language_code)
+def load_schema_file(schema_file, language_code=None):
+    """
+    Load a schema, optionally for a specified language.
+    :param language_code: ISO 2-character code for language e.g. 'en', 'cy'
+    """
+    language_code = language_code or 'en'
+    schema_path = _get_schema_file_path(schema_file, language_code)
 
-    try:
-        if language_code is not None and language_code != 'en':
-            schema_dir = os.path.join(settings.EQ_SCHEMA_DIRECTORY, language_code)
-        else:
-            schema_dir = settings.EQ_SCHEMA_DIRECTORY
+    if language_code != 'en' and not os.path.exists(schema_path):
```

Comment:
only in that the call would be identical to one above if the language code was 'en'